02LessonTwo/homework/index.html

```diff
@@ -16,13 +16,12 @@
 
```

Comment:
Thank you for the feedback! I've gone through and cleaned up the code, added a video/image and comments as well.